src/update_parameters.jl

```diff
@@ -282,5 +298,119 @@ function update_parameters!(model::Optimizer)
         end
     end
 
+    _update_vector_quadratic_constraints!(model)
```

Comment:
```suggestion
    # Different from the above cases, it is not possible to update
    # vector function in-place because there is no vector version of
    # MOI.ScalarCoefficientChange.
    # Therefore, vector quadratic function must be update via
    # MOI.ConstraintFunction that replaces the entire function.
    # Consequently, this operation is much slower.
    # Moreover, to perform the update, it is easier to use the
    # current_function code, hence, this update must be executed
    # after the update of the parameters cache (the loop above).
    _update_vector_quadratic_constraints!(model)
```